functions/webhooks/instagram/InstagramToFlex.ts

```diff
@@ -56,12 +56,15 @@ export type Body = InstagramMessageEvent & {
   bodyAsString?: string; // entire payload as string (preserves the ordering to decode and compare with xHubSignature)
 };
 
-const shouldFilterMessage = (message: InstagramMessageObject['message']) => {
-  // Filter story mention
-  if (message.attachments && message.attachments[0].type === 'story_mention') return true;
+const isMessageDeleted = (message: InstagramMessageObject['message']) => message.is_deleted;
 
-  return false;
-};
+const isStoryMention = (message: InstagramMessageObject['message']) =>
+  message.attachments && message.attachments[0].type === 'story_mention';
+
+const fromStoryMention = (message: InstagramMessageObject['message']) =>
```

Comment:
I was a little bit confused with this function being named `fromStoryMention`. It was not clear what it would return. Maybe `getStoryMentionText()` is a more readable name?